File: application/src/main/java/com/study/platform/domain/team/usecase/CreateTeamScheduleUseCase.java
package com.study.platform.domain.team.usecase;

import com.study.platform.domain.team.dto.request.TeamScheduleCreateRequest;
import com.study.platform.domain.team.dto.response.TeamScheduleResponse;

import java.util.UUID;

public interface CreateTeamScheduleUseCase {
    TeamScheduleResponse execute(UUID userId, UUID teamId, TeamScheduleCreateRequest request);
}

File: application/src/main/kotlin/com/study/platform/domain/team/usecase/CreateTeamScheduleUseCase.kt
package com.study.platform.domain.team.usecase

import com.study.platform.domain.team.dto.request.TeamScheduleCreateRequest
import com.study.platform.domain.team.dto.response.TeamScheduleResponse
import java.util.UUID

interface CreateTeamScheduleUseCase {
    fun execute(userId: UUID, teamId: UUID, request: TeamScheduleCreateRequest): TeamScheduleResponse
}
